tests/robottelo/test_func_locker.py

```diff
@@ -0,0 +1,321 @@
+# coding: utf-8
+
+from __future__ import with_statement
```

Comment:
yes there is a difference , in one test version I had a with statement in a try-except, and that is badly supported in 2.7, 
so will remove it as removed that code